src/writer/command_line.py

```diff
@@ -7,7 +7,7 @@
 import click
 
 import writer.serve
-from writer.deploy import cloud
+from writer.deploy import cloud, deploy_app
```

Comment:
Can't we keep only `deploy` method in `deploy.py` and use an alias here ?
```suggestion
from writer.deploy import cloud, deploy as deploy_app
```